modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -319,6 +319,8 @@ definition {
 		PortletEntry.publish();
 
 		WebContent.viewWithStructureCP(
+			webContentDate = "23/02/2021",
+			webContentDateFieldLabel = "Date - pt_BR",
```

Comment:
@guivilarim please notice that by doing this, you've duplicating the parameters _webContentDate_ and _webContentDateFieldLabel_